### DISCUSSION TITLE
Transfert mail old server to new server

### DISCUSSION THREAD

**Discussion by MarioC-IRL** [modoboa/modoboa#2282] [Help]
Hello, I currently have two modoboa servers and I am looking for a reliable solution to be able to transfer all the mails of a specific domain to the new server without losing any mail. I am looking for a reliable solution to transfer all the mails of a specific domain to the new server without losing any mails. The goal is to be able to migrate my domains without having any problem and to take advantage of the IPv6 compatibility of my new server

**Comment by tonioo** [comment]
Have you checked the IMAP migration plugin?

**Comment by MarioC-IRL** [comment]
> Have you checked the IMAP migration plugin?

No, is it a plugin included in modoboa?

**Reply by tonioo** [reply]
https://github.com/modoboa/modoboa-imap-migration
